envs/sourmash.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+   - conda-forge
+   - bioconda
+   - defaults
+dependencies:
```

Comment:
In the README for setup it just references the `environment.yml` and not the `sourmash.yml` or `tidyverse.yml` files for setup - this might be a snakemake thing I'm missing but how are these dependencies then integrated?

Reply:
Snakemake! in the rule, there's a `conda:` directive where you point to the path to the `sourmash.yml` or other yml files. Then snakemake uses conda to create these environments and then coordinates there activation for each rule.